tests/conftest.py

```diff
@@ -151,6 +161,19 @@ def storageclass_factory_class(
     )
 
 
+@pytest.fixture(scope='session')
+def storageclass_factory_session(
+    request,
+    ceph_pool_factory_session,
+    secret_factory_session
+):
+    return storageclass_factory_fixture(
+        request,
+        ceph_pool_factory_session,
+        secret_factory_session
+    )
```

Comment:
same

Reply:
@fbalak 
 Lets agree to use default CBP and secrets.. If required, we can create new SCs, but with default Secret and CBP

Reply:
Ok